module_bindings/README.md

```diff
@@ -0,0 +1,39 @@
+mv_referrable function bindings (used for complex types)
```

Comment:
Let's actually use markdown in this file. "Title" or "heading" sections can be marked with one or more leading `#` character (more hashes means nested as a sub-section). Also place a pair of back-ticks around in-line code, or code blocks with triple back-ticks before and after.